filters/ThirdParty/filter_255_EasyListPortuguese/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "filterId": 255,
```

Comment:
For EasyList filters used 100-199, so replace all 255.

Reply:
Done.